translate/api/TranslateV3Samples/TranslateV3BatchTranslateText.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START translate_v3_batch_translate_text]
+
+using System;
+using Google.Cloud.Translate.V3;
+
+namespace GoogleCloudSamples
+{
+    public static class TranslateV3BatchTranslateText
```

Comment:
All of these still have the TransalateV3 prefix. Consider removing. Both from the class name and file name.